sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -79,37 +79,59 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue or subscription.
+   *
+   * If the message is not settled during this time it becomes available to other receivers.
```

Comment:
It can become available to the same receiver as well...

> If the message is not settled during this time you will receive it again.

I know `will` is not completely right here as the message will get deadlettered after 10 tries, but that would be too much information?

> If the message is not settled during this time you will receive it again until maxDeliveryCount is reached. You can configure this count when creating the queue.

Reply:
How about this?

```javascript
  /* In peekLock mode, the receiver has a lock on the message for the duration specified on the
   * queue or subscription.
   *
   * Messages that are not settled will be redelivered as many times as the max delivery count
   * set on the queue or subscription.
   *
   * You can settle a message by calling complete(), abandon(), defer() or deadletter() methods on
   * the message.
   */
```

Reply:
(I'll fix the entity type thing like you mentioned below)